src/mixin/entity-mixin-lit.js

```diff
@@ -65,8 +69,9 @@ const InternalEntityMixinLit = superclass => class extends superclass {
 			const pendingEvent = new AsyncStateEvent(pendingPromise);
 			this.dispatchEvent(pendingEvent);
 
-			entityFactory(this._entityType, this.href, this.token, entity => {
+			entityFactory(this._entityType, this.href, this.token, (entity, error) => {
 				this._entity = entity;
+				this._error = error;
```

Comment:
To avoid any conflicts, I confirmed with [search.d2l](https://search.d2l.dev/search?full=EntityMixinLit&defs=&refs=&path=&hist=&type=&xrd=&nn=8&searchall=true) that no files which use `EntityMixinLit` make use of an `_error` property.

Reply:
Even so, it may be too generic and conflict in the future. Maybe `_entityError`?